src/foam/util/SafetyUtil.java

```diff
@@ -27,6 +27,10 @@ public static boolean equals(Object o1, Object o2) {
     return compare(o1, o2) == 0;
   }
 
+  public static boolean equalsIgnoreCase(String o1, String o2) {
```

Comment:
@kgrgreer do you want equalsIgnoreCase or equalsIC ? I know you like shorter names.

Reply:
IC is more compatible with mLangs but IgnoreCase is compatible with the Java function, which this once mimics, so I think the way it is is correct